tests/data/test_dataloader.py

```diff
@@ -549,10 +549,8 @@ def test_finetuning_dataloader_custom_split_remote(split: str):
 
 @pytest.mark.parametrize('pretokenize', [True, False])
 @pytest.mark.parametrize('use_multiple_streams', [True, False])
-@pytest.mark.parametrize('use_bytes', [True, False])
```

Comment:
The changes to the way streaming examples are formatted made this setting irrelevant

Reply:
I don't quite follow this. We still want to test the case where the dataset was written with bytes, since we support loading that for backwards compat, right?

Reply:
No, no, no. Maintaining backwards compatibility doesn't mean that you _also_ maintain test coverage of the previous functionality, duh.... wait, no, that doesn't sound right.... I should probably restore the test cases.